client/components/select-vertical/suggestion-search.tsx

```diff
@@ -134,12 +140,38 @@ const SelectVerticalSuggestionSearch: FC< Props > = ( {
 		] );
 	}, [ translate, suggestions, isLoading, isShowSuggestions, isShowSkipOption ] );
 
+	useEffect( () => {
+		if ( ! ( window.visualViewport && isMobile ) ) {
+			return noop;
+		}
+
+		const handleResize = () => {
+			if ( ! wrapperRef.current ) {
+				return;
+			}
+
+			wrapperRef.current.style.setProperty(
+				'--select-vertical-search-dropdown-height',
```

Comment:
hmm I don't understand why this is neccessary actually, wouldn't this be the equivalent of setting 40vh in scss. Doesn't that work on its own?

Reply:
The visual viewport is different from the layout viewport. User-interface features like the on-screen keyboard (OSK) can shrink the visual viewport without affecting the layout viewport. So, I think we need to update the height of the dropdown when the user is focusing on the input and the dropdown and on-screen keyboard shows.

See https://developer.mozilla.org/en-US/docs/Web/API/Visual_Viewport_API

Reply:
ahh thanks for explaining that!